Feature/nougat update

flibbertigibbet opened the pull request:
Updates to support Android Nougat (SDK 24).

WIP - opening PR to check Travis build.

flibbertigibbet committed:
Update libraries

Update libraries and target Nougat (SDK 24).

Updates form builder library for Nougat support.

flibbertigibbet committed:
Update CI configuration

Targeting Nougat.

ddohler reviewed (changes requested):
Looks good; +1 after travis success.

flibbertigibbet committed:
Use Java 8 on Travis

To fix UnsupportedClassVersionError, as new gradle version apparently was built with Java 8.

flibbertigibbet committed:
Convince Travis to update

flibbertigibbet committed:
Change Travis license acceptance regex

Rejecting all licenses now for unknown reasons.

flibbertigibbet committed:
Bump Travis memory

Switch to non-container instance for more memory.
